docs/src/main/paradox/sbt-plugin.md

```diff
@@ -0,0 +1,53 @@
+# sbt plugin details
+
+The sbt plugin is added to a project like so:
+
+@@@vars
+```scala
+// in project/plugins.sbt:
+addSbtPlugin("com.lightbend.akka.grpc" % "akka-grpc-sbt-plugin" % "$projectversion$")
+// in build.sbt:
+enablePlugins(AkkaGrpcPlugin)
+```
+@@@
+
+It will then by default look for `.proto` files under `src/main/protobuf`.
+
+## Configuration
+
+By default the plugin generates both a client and a server stub for Scala. 
+
+It can be configured to just generate either server or client like so:
+
+```scala
```

Comment:
Should we use Scala/Java tabs here?

Reply:
You mean for the generate-scala/java sources? I thought about that when writing it and that this is more for completeness in a complicated setup than two alternatives as it normally is with Java/Scala